.eslintrc.json

```diff
@@ -22,6 +23,8 @@
         "varsIgnorePattern": "^_",
         "caughtErrorsIgnorePattern": "^_"
       }
-    ]
+    ],
+    "@tanstack/query/exhaustive-deps": "warn",
```

Comment:
we can't set this one to `error` yet because there are some places where our contract instances are dependencies of our functions and would therefore be expected to be placed in the dependency array. This won't work because the instances are not json serializable. This is one of the problems that viem aims to fix, so I'm looking forward to moving over to that once its ready.